src/styles/utils/_loop.scss

```
@@ -0,0 +1,29 @@
+@for $i from 1 through 31 {
+  .calendar div:nth-child(#{$i})::after {
+    display: none;
+    content: "#{$i}";
+  }
+}
```

add just content, no need to make ;;after invisible by default - then you won't need to make it inline again

```suggestion
@for $i from 1 through 31 {
  .calendar div:nth-child(#{$i})::after {
    content: "#{$i}";
  }
}
```